apps/web/src/components/docs/DocsModal.tsx

```diff
@@ -77,18 +78,18 @@ export const DocsModal = ({ open, toggle, path }) => {
             })}
             gap="75"
           >
-            <ActionButton
-              tooltip="Open docs website"
-              onClick={() => {
-                openInNewTab(`${DOCS_URL}/${path}`);
-              }}
-              Icon={() => <IconOpenInNew />}
+            <IconButton
+              as="a"
+              href={`${DOCS_URL}/${path}`}
+              target="_blank"
+              rel="noreferrer noopener"
+              Icon={IconOpenInNew}
```

Comment:
Better to use href for `copy link` functionality.

Reply:
Why do we open docs here and not the modal?

Reply:
This is the modal button, shown in screenshot
<img width="756" alt="image" src="https://github.com/novuhq/novu/assets/32132657/210b68bc-418f-4341-954a-22afe96754e6">